plugins/wmap.rb

```diff
@@ -1416,7 +1416,7 @@ def process_urls(urlstr)
         end
 
         #print_status "aaa"
-        #print_status framework.db.workspace.name
+        print_status framework.db.workspace.name
```

Comment:
Was this uncommenting intentional?

Reply:
Excuseme,the problem how to slove?

Reply:
I pull head master now.But,this problem not slove.